README.md

```diff
@@ -74,6 +74,8 @@ yaml-validator [options] <file>
 Version 1.1.0
 ```
 
+When used from the command-line, the process exits with the count of invalid files.
```

Comment:
Please use `number of` as oppose to `count of`. A bit of background https://www.enago.com/academy/understanding-the-difference-between-number-and-amount/

Reply:
Fixed